src/parser/view_transform.c

```diff
@@ -3147,6 +3153,45 @@ pt_copypush_terms (PARSER_CONTEXT * parser, PT_NODE * spec, PT_NODE * query, PT_
 	    }
 	}
       break;
+    case PT_DBLINK_TABLE:
+      /* copy terms */
+      query->info.dblink_table.pushed_pred = parser_copy_tree_list (parser, term_list);
+      save_custom = parser->custom_print;
+      parser->custom_print |= PT_CONVERT_RANGE | PT_SUPPRESS_RESOLVED | PT_PRINT_NO_HOST_VAR_INDEX;
+      pushed_pred = pt_print_and_list (parser, query->info.dblink_table.pushed_pred);
+
+      /* wrapped query SELECT * FROM */
+      rewritten = pt_append_bytes (parser, rewritten, "SELECT * FROM (", 15);
+      query_str = query->info.dblink_table.qstr->info.value.data_value.str;
+      rewritten = pt_append_varchar (parser, rewritten, query_str);
+
+      /* alias name : '_r' */
+      rewritten = pt_append_bytes (parser, rewritten, ") AS _r", 7);
```

Comment:
alias name `_r` possibly conflict?

Reply:
it's no conflict because the sub-query is not depend on the main query. it's used just as remote query at the remote server.